plugins/bitbucket-cloud-common/package.json

```diff
@@ -42,11 +42,13 @@
   },
   "dependencies": {
     "@backstage/integration": "workspace:^",
-    "cross-fetch": "^4.0.0"
+    "cross-fetch": "^4.0.0",
```

Comment:
Are these references needed here since we now reference the FetchService in the integration dependency?

Reply:
This has been removed

Reply:
Removing these broke the build. I'm putting them back.